integration_tests/plugins/test_ruby_plugin.py

```diff
@@ -33,6 +33,14 @@ def test_ruby_gem_install_rack(self):
 
     def test_ruby_hello(self):
         self.run_snapcraft('stage', 'ruby-hello')
+        os.environ['RUBYPATH'] = os.path.join(self.stage_dir, 'bin')
```

Comment:
@kalikiana I did this because I couldn't find any other way to pass stage_dir root to create the env vars needed by ruby.

Do you know a better way to do this?

https://forum.snapcraft.io/t/plugins-setting-env-vars-for-stage-test/1343